lib/sanbase/social_data/social_dominance.ex

```diff
@@ -61,6 +63,34 @@ defmodule Sanbase.SocialData.SocialDominance do
     end
   end
 
+  def social_dominance_trending_words() do
+    now = Timex.now()
+    from = Timex.shift(now, hours: -@hours_back_ensure_has_data)
+    to = now
+    interval = "1h"
+    source = :total
+
+    with {:ok, trending_words} <-
+           Sanbase.SocialData.TrendingWords.get_currently_trending_words(@trending_words_size),
+         words <- Enum.map(trending_words, & &1.word),
+         {:ok, words_volume} <-
+           Sanbase.SocialData.social_volume(%{words: words}, from, to, interval, source),
+         {:ok, total_volume} <-
+           Sanbase.SocialData.social_volume(%{text: "*"}, from, to, interval, source) do
+      words_mentions_sum =
+        words_volume
+        |> Enum.reduce(0.0, fn word, acc ->
+          acc + List.last(word.timeseries_data).mentions_count
+        end)
```

Comment:
In this case we know the list has at most 3 elements, so `reduce` would not bring any performance benefits. I think with 3 functions it would be easier to read.
```suggestion
      words_mentions_sum =
        Enum.map(words_volume, &List.last(&1).mentions_count)
        |> Enum.sum()
```